invenio_app_ils/errors.py

```diff
@@ -213,6 +213,17 @@ def __init__(self, document_pid, **kwargs):
         self.description = self.description.format(document_pid)
 
 
+class LibraryNotFoundError(IlsException):
```

Comment:
Sorry, can you move this inside the ILL package/module? we tried to keep ILL/ACQ isolated and independent as much as possible. Thanks!